### PR TITLE
Create an OpenFisca key pass

### LINKED ISSUE #36: Create an OpenFisca key pass
Relates to #27

### PR DESCRIPTION
**Pull request by maukoquiroga**
Fixes #36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,5 +68,25 @@ service nginx reload
...
+
+Access to shared services (OVH, Twitter, etc.) must be done through individual accounts, to which the management is delegated.
+
+However, some systems do not support delegation, or it can sometimes be helpful to get administrator access to master accounts.
+
+A database containing all the necessary passwords is therefore accessible.
+
+### Instructions
+
+* Install [KeePassX](https://www.keepassx.org)
```
Comment (sandcha) [inline README.md]: >Update link to [downloads page](https://www.keepassx.org/downloads)?

File: README.md [hunk 2]
```diff
@@ -69,5 +68,25 @@ service nginx reload
...
+* Download the [database](openfisca.kdbx)
```
Comment (sandcha) [inline README.md]: Add the implicit step where the user should run KeePassX and follow `Database` > `Open database`?

File: README.md [hunk 3]
```diff
@@ -69,5 +68,25 @@ service nginx reload
...
+* Request key and password on the [#of-ops](https://openfisca.slack.com) channel.
```
Comment (sandcha) [inline README.md]: So, the targeted user must have access to this slack. Explain earlier who is this documentation for?
> OpenFisca France community administrators? 🤔

File: README.md [hunk 4]
```diff
@@ -69,5 +68,25 @@ service nginx reload
...
+
+The underlying intention is to reduce the risk of loss of control over password-protected components, and to provide, in the long-term, a standard access to OpenFisca’s services and accounts, in order to improve delivery, fail safety and resilience.
```
Comment (sandcha) [inline README.md]: > Remove one comma or split this line after `accounts`? ^^

File: README.md [hunk 5]
```diff
@@ -69,5 +68,25 @@ service nginx reload
...
+
+Do not transmit key and password through the same channel. Password must be communicated orally, and key should be transmitted either by physical transfer (USB key, etc.) or electronically, in which case you’ll have to encrypt it before sending.
+
+Currently, access to both key and password are granted to @Anna-Livia, @maukoquiroga and @sandcha. This is expected to change as the community sees the need.
```
Comment (sandcha) [inline README.md]: Add links to accounts?

File: README.md [hunk 6]
```diff
@@ -17,7 +17,7 @@ For more information on `auto-update` sub-directories, please see their own READ
 
 ## Server configuration
 
-The nginx configuration files are loaded from the `/etc/nginx/sites-available` directory.  
+The nginx configuration files are loaded from the `/etc/nginx/sites-available` directory.
```
[the remark this answers is not in the document]
Comment (maukoquiroga) [reply README.md]: There's no new line change here, just the removal of spaces.

File: README.md [hunk 7]
```diff
@@ -29,7 +29,7 @@ service nginx reload
 
 We use [systemd](https://wiki.debian.org/systemd) services to ensure the continued operation of the OpenFisca related applications (OpenFisca-France [web API](https://fr.openfisca.org/api/v18), [legislation explorer](https://legislation.openfisca.fr/)). `systemd` is available on most recent (> 2015) linux distributions.
 
-The service configuration files are loaded from the `/etc/systemd/system` directory.  
+The service configuration files are loaded from the `/etc/systemd/system` directory.
```
[the remark this answers is not in the document]
Comment (maukoquiroga) [reply README.md]: Same thing.

File: README.md [hunk 8]
```diff
@@ -69,5 +68,25 @@ service nginx reload
...
-- create a new service with a symlink 
-- reload the daemon 
+- create a new service with a symlink
+- reload the daemon
+
+## Passwords
+
+Access to shared services (OVH, Twitter, etc.) must be done through individual accounts, to which the management is delegated.
+
+However, some systems do not support delegation, or it can sometimes be helpful to get administrator access to master accounts.
```
[the remark this answers is not in the document]
Comment (maukoquiroga) [reply README.md]: What do you mean ? There are three lines : Access to... however... therefore. I'd break narrative logic if I sweep lines.

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (maukoquiroga) [reply README.md]: There never was a new line there.

File: README.md [hunk 9]
```diff
@@ -69,5 +68,25 @@ service nginx reload
...
+
+A database containing all the necessary passwords is therefore accessible.
+
+### Instructions
```
[the remark this answers is not in the document]
Comment (maukoquiroga) [reply README.md]: Isn't it already implicit given the context?

File: README.md (hunk 1 above)
Comment (maukoquiroga) [reply README.md]: OK

File: README.md (hunk 2 above)
Comment (maukoquiroga) [reply README.md]: Isn't it up to the user ? I'm just explaining how to get access to the passwords database.

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (sandcha) [reply README.md]: two spaces = new line 🙂

File: README.md (hunk 2 above)
Comment (sandcha) [reply README.md]: We have two steps and only an implicit link between them. If it's a user guide, I would add the third step (run KeePassX and follow Database > Open database). If it's just about where to find the database, it's optional.

File: README.md (hunk 3 above)
Comment (sandcha) [reply README.md]: 😢

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (maukoquiroga) [reply README.md]: Ah, the thing is I'll have to deactivate the automatic space remover. Why two spaces and not two newlines?

**Pull request merged**